website/content/docs/commands/var/lock.mdx

```diff
@@ -0,0 +1,109 @@
+---
+layout: docs
+page_title: "Command: var lock"
+description: |-
+  The "var lock" command locks a variable, only allowing the lock owner to modify it.
+---
+
+# Command: var lock
+
+The `var lock` command holds a lock on a [variable][].
+
+## Usage
+
+```plaintext
+nomad var lock [options] <lock spec file reference> child...
+nomad var lock [options] <path to store variable> [<variable spec file reference>] child
+```
+
+The lock command provides a mechanism for simple distributed locking. A lock
+is created in the given variable, and only when held, is a child process invoked.
+
+The lock command can be called on an existing variable or an entire new variable
+specification can be provided to the command from a file by using an
+@-prefixed path to a variable specification file. Items to be stored in the 
+variable can be supplied using the specification file as well. 
+
+Nomad lock launches its children in a shell. By default, Nomad will use the
+shell defined in the environment variable SHELL. If SHELL is not defined, 
+it will default to /bin/sh. It should be noted that not all shells terminate
+child processes when they receive SIGTERM. Under Ubuntu, /bin/sh is linked 
+to dash, which does not terminate its children. In order to ensure that 
+child processes are killed when the lock is lost, be sure to set the SHELL
+environment variable appropriately, or run without a shell by setting -shell=false. 
+ 
+If ACLs are enabled, this command requires the 'variables:write' capability
```

Comment:
Consider linking to https://developer.hashicorp.com/nomad/tutorials/variables/variables-acls here

Reply:
Will do!